Sources/Latte+MigrateSession.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+public struct MigratedSession: Decodable {
+    let tokenType: String?
+    let accessToken: String?
+    let expiresIn: Int?
+    let refreshToken: String?
+}
+
+public extension Latte {
+    func migrateSession(
+        accessToken: String,
+        migrationEndpoint: String,
```

Comment:
We should make a breaking change in the SDK.

1. (Breaking) Rename `tokenizeEndpoint` to `middlewareEndpoint`
2. The SDK derives tokenizeEndpoint and sessionMigrationEndpoint from `middlewareEndpoint`

Reply:
Updated at [5d1976f](https://github.com/authgear/authgear-sdk-ios/pull/183/commits/5d1976f5cc303490e1f5cbdd56394f6cce48c4ca)